### PR TITLE
welcome: successful registration page

### PR DESCRIPTION
**Pull request by giolekva**

### CODE REVIEW

File: core/installer/welcome/welcome.go [hunk 1]
```diff
@@ -27,19 +30,22 @@ type Server struct {
...
 	createAccountAddr string
+	loginURL          string
 }
 
 func NewServer(
 	port int,
 	repo installer.RepoIO,
 	nsCreator installer.NamespaceCreator,
 	createAccountAddr string,
+	loginURL string,
```
Comment (giolekva) [inline core/installer/welcome/welcome.go:41]: Did you try running `make build` or `push`?
This won't compile as caller of the `NewServer` was not updated to pass `loginURL`

File: core/installer/welcome/welcome.go [hunk 2]
```diff
@@ -27,19 +30,22 @@ type Server struct {
 	repo              installer.RepoIO
 	nsCreator         installer.NamespaceCreator
 	createAccountAddr string
+	loginURL          string
```
Comment (giolekva) [inline core/installer/welcome/welcome.go:33]: rename `loginURL` to `loginAddr` to be consistent with `createAccountAddr` variable

File: core/installer/welcome/registration-success.html [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+        <link rel="stylesheet" href="/static/pico.2.0.6.min.css">
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1" >
+        <title>Successful Registration</title>
+    </head>
+    <body>
+        <main class="container">
+            <div>
+                <p>Your account has been successfully created.</p>
+                <p>Click <a href="{{ .LoginURL }}">here</a> to log in.</p>
```
Comment (giolekva) [inline core/installer/welcome/registration-success.html:13]: Replace `to log in` with `to authenticate`

File: core/installer/welcome/registration-success.html
Comment (giolekva) [inline core/installer/welcome/registration-success.html:1]: rename this file to `create-account-success.html` so all html files related to new account are named similarly.

File: core/installer/welcome/welcome.go (hunk 2 above)
Comment (DTabidze) [reply core/installer/welcome/welcome.go:33]: Done

File: core/installer/welcome/registration-success.html (hunk 3 above)
Comment (DTabidze) [reply core/installer/welcome/registration-success.html:13]: Done

File: core/installer/welcome/registration-success.html
Comment (DTabidze) [reply core/installer/welcome/registration-success.html:1]: Done

File: core/installer/welcome/welcome.go (hunk 1 above)
Comment (DTabidze) [reply core/installer/welcome/welcome.go]: Done.

File: core/installer/cmd/welcome.go [hunk 4]
```diff
@@ -1,9 +1,10 @@
 package main
 
 import (
-	"golang.org/x/crypto/ssh"
 	"os"
 
+	"golang.org/x/crypto/ssh"
+
```
Comment (giolekva) [inline core/installer/cmd/welcome.go:7]: rm empty line

File: core/installer/cmd/welcome.go [hunk 5]
```diff
@@ -47,6 +49,12 @@ func welcomeCmd() *cobra.Command {
 		"",
 		"",
 	)
+	cmd.Flags().StringVar(
```
Comment (giolekva) [inline core/installer/cmd/welcome.go:52]: Good, now you'll have to actually pass the value to this flag. You'll need to update following files:
https://github.com/giolekva/pcloud/blob/3183056f2cb91eca8d0b182fc56169af0033c750/charts/welcome/templates/install.yaml#L112
https://github.com/giolekva/pcloud/blob/3183056f2cb91eca8d0b182fc56169af0033c750/charts/welcome/values.yaml#L7
https://github.com/giolekva/pcloud/blob/3183056f2cb91eca8d0b182fc56169af0033c750/core/installer/values-tmpl/welcome.cue#L40

Login address will be: `http://accounts-ui.\(global.domain)`

Comment (DTabidze) [reply core/installer/cmd/welcome.go:50]: Done

File: core/installer/cmd/welcome.go (hunk 4 above)
Comment (DTabidze) [reply core/installer/cmd/welcome.go]: Done

**Pull request merged**